javascript-loops/main.js

```diff
@@ -0,0 +1,71 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+  while (count <= times) {
+    repeated += word + ' ';
+    count++;
+  }
+  return repeated;
+}
+
+function logEachCharacter(string) {
+  let i = 0;
+  for (i = 0; i < string.length; i++) {
```

Comment:
You can also just declare `i` inside the `for` loop. `for (let i = 0; i < string.length; i++)`